### DISCUSSION TITLE
Comments in MyST that don't break paragraphs

### DISCUSSION THREAD

**Discussion by alimanfoo** [executablebooks/meta#502] [Ideas] [answered]
Hi folks,

I'd like to be able to use [comments](https://myst-parser.readthedocs.io/en/latest/syntax/syntax.html#comments) within MyST, but currently comments are a block-level entity, which means they cause the start of a new paragraph. I'd find it very useful if there was some way to write comments which didn't break up paragraphs. 

Just thought I'd float the idea here, is this worth raising an issue on the myst-parser repo?

**Comment by chrisjsewell** [comment, marked as answer]
Heya, there is already an issue for this https://github.com/executablebooks/MyST-Parser/issues/116 (give the initial comment a 👍 to upvote)

**Reply by alimanfoo** [reply]
Awesome, thanks!
